src/plots/areaPlot.ts

```diff
@@ -119,6 +118,10 @@ export class Area<X> extends Line<X> {
   }
 
   protected _additionalPaint() {
+    if (this.renderer() === "canvas") {
```

Comment:
We should configure the additional drawstep to use canvas to draw the line (that achieves the same goal as the "draw step composition") and delete the copy/pasted code in area renderer